app/enterprise/0.35-x/kong-manager/authentication/super-admin.md

```diff
@@ -29,4 +29,18 @@ click the email address of the user in order to view them.
 
 5. Click "Generate Registration Link". 
 
-6. Copy the link for later use after completing the account setup.
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
```

Comment:
Relative paths do not reflect the structure of how the site is built, so it 404s

```suggestion
In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](/enterprise/{{page.kong_version}}/getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
```